### ISSUE TITLE
Issues on SynchronizeRealtime

### ISSUE THREAD

**Issue by tbeu** [modelica/Modelica_DeviceDrivers#42]
MDD_realtimeSynchronize uses static variables which is no good programming since it is not thread-safe. Therefore it would be better to implement Modelica_DeviceDrivers.Blocks.OperatingSystem.SynchronizeRealtime as an external object.

**Comment by tbeu** [comment]
Did you manage to set up a Win installation. If not I can offer to test it.

**Comment by bernhard-thiele** [comment]
I have a windows installation, just no VS2010 installed on it (only VS2013) so that I can't build Windows XP compatible libraries as you explained to me.
However, I have merged all of your last patches in Linux and not yet tested for Windows. So if you want to test it, you are welcome :-).

**Comment by tbeu** [comment]
Which branch is it? I do not find any changes that remove static variables from MDD_realtimeSynchronize.

**Comment by bernhard-thiele** [comment]
I thought I had pulled a patch from you that solved it. However, I just searched the history and the patches that I pulled from you addressed other things
(6d2830a4d385eae47cc6bc3b18743242dec089fb, 6d2830a4d385eae47cc6bc3b18743242dec089fb, 6d2830a4d385eae47cc6bc3b18743242dec089fb)
Is there a patch that I missed?

If not, no problem I will change it to external objects. Just not sure if I find the time today.
